README.md

```diff
@@ -78,14 +78,15 @@ The service must also provide a callbackUrl for asynchronous notification of the
 (Although this rule is relaxed when testing locally with [upscan-stub](https://github.com/hmrc/upscan-stub) rather than [upscan-initiate](https://github.com/hmrc/upscan-initiate).
 In this stubbed scenario a `callbackUrl` referring to localhost may still specify `http` as the protocol.)
 
-Session-ID / Request-ID headers will be used to link the file with the user's journey.
+Session-ID / Request-ID headers are used to link the file with the user's journey.
 
 *Note:* If you are using `[http-verbs](https://github.com/hmrc/http-verbs)` to call Upscan, all the headers will be set automatically
 (See: [HttpVerb.scala](https://github.com/hmrc/http-verbs/blob/2807dc65f64009bd7ce1f14b38b356e06dd23512/src/main/scala/uk/gov/hmrc/http/HttpVerb.scala#L53))
 
 The service replies with a pre-filled template for the upload of the file.
-The JSON response also contains a globally unique file reference for the upload. This reference can be used by the Upscan service team to view the progress and result of the journey through the different Upscan components. The consuming service can use this reference to correlate the subsequent upload result with this upscan initiation.
-
+The JSON response contains a globally unique identifier for the upload (known as both _reference_ and _key_).
+This identifier can be used by the Upscan service team to view the progress and result of the journey through the different Upscan components. 
```

Comment:
Is it worth mentioning something to the effect of - if there's an issue and that requires further investigation we require the reference.

Reply:
Yes - added